setup/productionize/scaling/horizontal-scaling.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: single
+title:  "Horizontally Scale Spinnaker Services"
+sidebar:
+  nav: setup
+---
+
+Spinnaker's services are intended to be replicated for performance and
+availability. This page will explain under which circumstances it can help
```

Comment:
Does this break the meaning?: 
"You can scale Spinnaker for performance and availability by replicating its microservices."

Reply:
No, will reword

Reply:
Actually it sort-of does... what about:

"You can improve Spinnaker's performance and availability by scaling its microservices"